site/content/entry/yuzu-progress-report-apr-2020/index.md

```diff
@@ -0,0 +1,208 @@
+    "./31.png| Performance may be reduced with more accurate levels"
+  >}}
+
+{{< single-title-imgs
+    "Normal accuracy (left) and High accuracy (right) (Super Mario Odyssey)"
+    "./v1.mp4"
+    "./v2.mp4"
+  >}}
+
+
+## One Line to Fix 'em All
+
+`Toki Tori` had sound problems that were related to a floating point instruction, and while investigating the issue, 
+[Merry](https://github.com/MerryMage) and [bunnei](https://github.com/bunnei) were worried that it may be a problem in [Dynarmic](https://github.com/MerryMage/dynarmic), our ARM to x64 recompiler.
+Turns out the problem was just a bad default value in yuzu’s Dynarmic configuration.
+Remember what we said about bunnei and serendipity? With a simple one line change, we fixed tons of bugs on a lot of games! Special mention goes to `Pokémon Sword/Shield`, where thanks to this single line change, the game no longer softlocked and became fully playable!
+
+Here's a *partial* list of fixes:
+
++ Pokémon Sword/Shield - All softlocks fixed, running in tall grass fixed, hair salons fixed
++ The Legend of Zelda: Link's Awakening - Miniature placing fixed
++ Toki Tori, Final Fantasy 7, Diablo 3, Project DIVA MegaMix - Distorted audio fixed
++ Team Sonic Racing - Physics fixed
++ Onimusha Warriors, Tales of Vesperia, The Messenger - Game logic fixed
++ Skyrim - Audio looping fixed, game progresses further
++ Oninaki - Elevator softlock fixed
++ All Unreal Engine 4 games - Motion jitter fixed
++ Super Smash Bros. Ultimate - Fighter select jitter / flickers fixed
++ Starlink - Softlock fixed, game progresses further
+
+&nbsp;
+
+{{< imgs
+    "./32.png| This sofa is no longer our enemy (Pokémon Sword/Shield)"
+  >}}
+
+## Bug fixes and improvements
+[Morph](https://github.com/Morph1984) did some Kirby magic and implemented the first steps to get World of Light 
+working in `Super Smash Bros. Ultimate`. 
+Even thou it doesn’t work yet, the first step is always the most important one!
+
+[Kewlan](https://github.com/Kewlan) once again strikes with more UI quality of life improvements. 
+This time, more UI fixes to the placement of buttons mappings in the input settings, specifically this time for single joycons.
+
+{{< imgs
+    "./03.png| From this"
+  >}}
+  
+{{< imgs
+    "./05.png| To this"
+  >}}
+
+Not stopping there, Kewlan added a slider for the analog stick modifier, allowing keyboard users to half press the emulated 
+sticks. Plus now the sliders show percentages, making them easier to read.
+
+{{< imgs
+    "./07.png| Precision"
+  >}}
+
+And to top it off, he also added a way to restore the default value for a hotkey. Part of emulation is having a tidy interface 
+to set everything up.
+
+{{< imgs
+    "./08.png| More customization"
+  >}}
+
+[Tobi](https://github.com/FearlessTobi) implemented some fixes to the internal working of the input settings and the game list, 
+besides that, he also changed some of our error and warning windows to something more user friendly, thanks Tobi!
```

Comment:
Forgot about:

I'd replace the sentence with the below:

`[Tobi](https://github.com/FearlessTobi) fixed some of the internal workings of the input settings and games list, improving the loading time of the games list exponentially. He also reworded some of our error and warning popups to be more user friendly.`

I also removed the `thanks Tobi` as it seemed out of place.